pithos/pandora/pandora.py

```diff
@@ -89,6 +89,40 @@ class ApiError(IntEnum):
     # Catch all for undocumented error codes
     UNKNOWN_ERROR = 100000
 
+    @property
+    def title(self):
+        # Turns RANDOM_ERROR into Pandora Error: Random Error
+        return 'Pandora Error: {}'.format(self.name.replace('_', ' ').title())
+
+    @property
+    def sub_message(self):
+        value = self.value
+        if value == 1:
+            return 'Pandora is performing maintenance.\nTry again later.'
+        elif value == 12:
+            return 'Pandora is not available in your country.\n
```

Comment:
__[flake8]__
* [E713] test for membership should be 'not in'
* [E901] SyntaxError: EOL while scanning string literal

<!-- Comment by [SideCI](https://sideci.com) -->